README.md

```diff
@@ -57,6 +57,12 @@ MyApplication::Application.config.session_store :redis_store, {
   key: "_#{Rails.application.class.parent_name.downcase}_session"
 }
 ```
+(**NOTE:** You can not write `:expires_in` or `:expire_in`as a substitute for `:expire_after`.)
```

Comment:
I'd rather the language here said something along the lines of "Please use `:expire_after` for session store configuration and `:expires_in` for Rails cache store configuration.", because right now it looks like you're making a general statement about the different key names, and that's kinda confusing.